ProjectCoimbra.UWP/Project.Coimbra/Pages/SongPage.xaml.cs

```diff
@@ -46,5 +48,34 @@ private async void NextButton_Click(object sender, RoutedEventArgs e)
 
             this.Next.IsEnabled = false;
         }
+
+        private async void Remove_Item(object sender, RoutedEventArgs e)
+        {
+            ContentDialog dialog = new ContentDialog
+            {
+                Title = "Remove Song from List",
```

Comment:
These strings should be taken from the resources file rather than being hard coded in